doc/man3/BIO_meth_new.pod

```diff
@@ -133,6 +133,20 @@ the L<BIO_callback_ctrl(3)> page for more information. This function will be cal
 in response to the application calling BIO_callback_ctrl(). The parameters for
 the function have the same meaning as for BIO_callback_ctrl().
 
+=head1 RETURN VALUES
+
+BIO_get_new_index() returns the new BIO type value or -1 if an error occurred.
+
+BIO_meth_new(int type, const char *name) returns a valid B<BIO_METHOD> or NULL
+if an error occurred.
+
+BIO_meth_free() does not return a value.
+
+The B<BIO_meth_set> functions return 1 on success. Currently these functions
+can return 1 only.
```

Comment:
I think we should document an error value of 0 even if the code doesn't currently return it. That gives us the ability to return an error in the future if we want to.